docs/src/basics/exporters.md

```diff
@@ -0,0 +1,89 @@
+# Exporters
+
+As Zeebe processes jobs and workflows, or performs internal maintenance (e.g.
+raft failover), it will generate an ordered stream of records:
+
+![record-stream](/basics/exporters-stream.png)
+
+While the clients provide no way to inspect this stream directly, Zeebe can load
+and configure user code that can process each and every one of those records, in the form of an exporter.
+
+An **exporter** provides a single entry point to process every record that is
+written on a stream.
+
+With it, you can:
+
+* Persist historical data by pushing it to an external data warehouse
+* Export records to a visualization tool (e.g. [zeebe-simple-monitor](https://github.com/zeebe-io/zeebe-simple-monitor/))
+
+You can find a reference exporter implementation in the form of the Zeebe-maintained
+[ElasticSearch exporter](https://github.com/zeebe-io/zeebe/tree/{{commit}}/exporters/elasticsearch-exporter).
+
+For more information, you can read the [reference information page](reference/exporters.html).
+
+> **Note**: it is not recommended to use exporters for orchestration purposes, as
+> that is what workflows are for.
+
+## Usage
+
+Zeebe will only load exporters which are configured through the main Zeebe TOML
+configuration file.
+
+Exporters are configured as TOML maps, with a standard preamble describing the
+exporter:
+
+```toml
+[[exporters]]
+id = "myExporter"
+className = "com.myCompany.MyExporter"
+```
+
+Optionally, an exporter can be loaded through an external JAR file, simply by
+pointing to it through the configuration:
+
+```toml
+[[exporters]]
+id = "myExporter"
+className = "com.myCompany.MyExporter"
+jarPath = "exporters/com.myCompany.exporters.jar"
+```
+
+Additionally, you can configure exporter-specific arguments through a nested map
+under the label `[exporters.args]`:
+
+```toml
+[[exporters]]
+id = "myExporter"
+className = "com.myCompany.MyExporter"
+jarPath = "exporters/com.myCompany.exporters.jar"
+  [exporters.args]
+  username = "mmustermann"
+  host = "udp://myCompany.com:8080"
```

Comment:
Here I asked myself how I would access the configuration. I think that also gets clearer if this section goes into Reference -> Exporters.